src/main/java/uk/gov/ons/ctp/response/action/export/service/ManifestBuilder.java

```diff
@@ -41,15 +41,27 @@ private PrintFileMainfest createManifest(String filename, ByteArrayOutputStream
 
     String manifestCreatedDateTime = DateTimeFormatter.ISO_INSTANT.format(Instant.now());
 
-    // These are hardcoded for now, they will be update and passed in from the actionsvc in the
-    // future
-    return new PrintFileMainfest(
-        1,
-        files,
-        "ONS_RM",
-        manifestCreatedDateTime,
-        "Initial contact letter households - England",
-        "PPD1.1",
-        1);
+    PrintFileMainfest pfm =
+        new PrintFileMainfest(
+            1,
+            files,
+            "ONS_RM",
+            manifestCreatedDateTime,
+            "Initial contact letter households - England",
+            "PPD1.1",
+            1);
+
+    if (filename.startsWith("P_IC_ICL2")) {
```

Comment:
I guess this works, but it's not the most elegant bit of code, throwing away the first object we created. Not sure if I'm being fussy, but should we just build the string we need rather than the whole object?